asreview/webapp/src/App.js

```diff
@@ -65,10 +66,22 @@ const App = () => {
   // Settings hook
   const [fontSize, handleFontSizeChange] = useFontSize();
 
-  const muiTheme = createTheme({
-    // cssVariables: true,
-    colorSchemes: { dark: true },
+  const [mode, setMode] = React.useState(() => {
+    const savedMode = localStorage.getItem("themeMode");
+    return savedMode || "system";
   });
+
+  React.useEffect(() => {
+    localStorage.setItem("themeMode", mode);
+  }, [mode]);
+
+  const prefersDarkMode = useMediaQuery("(prefers-color-scheme: dark)");
+
+  const muiTheme = React.useMemo(
+    () => getTheme(mode, prefersDarkMode),
+    [mode, prefersDarkMode],
+  );
+
```

Comment:
I think we should stick to the new colorSchemes API of MUI6. As far as I understand, there is no need to interact with the localStorage. See https://mui.com/material-ui/customization/palette/#color-schemes and https://mui.com/material-ui/customization/dark-mode/#system-preference